USAGE_android.md

```diff
@@ -306,6 +306,9 @@ option values.
 | Capture trigger for Android                    | debug.gfxrecon.capture_android_trigger                        | BOOL    | Set during runtime to `true` to start capturing and to `false` to stop. If not set at all then it is disabled (non-trimmed capture). Default is not set.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| Use asset file                                 | debug.gfxrecon.capture_use_asset_file                         | BOOL    | When set to `true` assets (images, buffers and descriptors) will be stored separately into an asset file instead of the capture file.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
+| Use asset file                                 | debug.gfxrecon.capture_android_dump_assets                         | BOOL    | Setting this triggers a dump of all assets into the asset file. Since android options cannot be set by the layer, dumping is done whenever this option switches between from `false` to `true` or from `true` to `false`. Default is: `false`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
```

Comment:
should this be "Dump asset file" instead of "Use asset file"?

Reply:
I also wonder if this should only trigger on the false -> true setting?

Reply:
> should this be "Dump asset file" instead of "Use asset file"?

Right, copy paste mistake

> I also wonder if this should only trigger on the false -> true setting?

Initially that was my idea too but then I realized that this is problematic. If an asset dump is triggered by the android property, once the assets are dumped the property must be transitioned to false, otherwise it will keep dumping every frame. The option needs to be cleared by the application but, at least as far as I can tell, android apps do not have permission to do that.